scripts/apps/authoring/authoring/components/text-statistics.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import {CharacterCount2} from 'apps/authoring/authoring/components/CharacterCount';
+import {gettextPlural} from 'core/utils';
+import {countWords} from 'core/count-words';
+import {getReadingTimeText} from 'apps/authoring/authoring/directives/ReadingTime';
+
+interface IProps {
+    text: string;
+    language?: string;
+    limit?: number;
+}
+
+export class TextStatistics extends React.PureComponent<IProps> {
+    render() {
+        const wordCount = countWords(this.props.text);
+        const readingTime: string = getReadingTimeText(this.props.text, this.props.language);
+
+        return (
+            <div style={{display: 'flex', gap: '6px'}}>
+                <span className="char-count-base">
+                    {gettextPlural(wordCount, 'one word', '{{x}} words', {x: wordCount})}
+                </span>
+
+                <CharacterCount2
```

Comment:
Can you rename this for better readability? How does it differ with `CharacterCount`?

Reply:
> How does it differ with CharacterCount?

Jump to definition and see the comment.